src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -814,6 +814,7 @@ var TorrentsTable = new Class({
                 case "checkingUP":
                 case "queuedForChecking":
                 case "checkingResumeData":
+                    case "moving":
```

Comment:
indentations are off in this file

Reply:
Sorry. It was a rebase conflict which was auto-solved.
I'll fix it later.